spec/unit/recipes/github_comment_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../../spec_helper'
+
+describe 'osl-jenkins::github_comment' do
+  ALL_PLATFORMS.each do |p|
+    context "#{p[:platform]} #{p[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(p) do |node|
+          node.set['osl-jenkins']['credentials']['git'] = {
+            'github_comment' => {
+              user: 'manatee',
+              token: 'token_password'
+            }
+          }
+          node.set['osl-jenkins']['credentials']['jenkins'] = {
+            'github_comment' => {
+              user: 'manatee',
+              pass: 'password',
+              trigger_token: 'trigger_token'
+            }
+          }
+        end.converge(described_recipe)
+      end
```

Comment:
You need to add ``include_context 'common_stubs'`` so that these tests now run properly.